gradoop-flink/src/test/java/org/gradoop/flink/model/impl/operators/transformation/TransformationTest.java

```diff
@@ -33,7 +33,7 @@
 
 public class TransformationTest extends GradoopFlinkTestBase {
 
-  protected static final String TEST_GRAPH = "" +
+  static final String TEST_GRAPH = "" +
```

Comment:
Why that change from `protected` to `static`? I prefer package private :)

Reply:
It was static before. I'll change it to a non-static attribute